### ISSUE TITLE
Visual Cue for Current Scheduled Task 

### ISSUE THREAD

**Issue by vinaybadgujar102** [tijlleenders/ZinZen#1775]
**Description**
Introduce a visual cue for tasks scheduled for the current time. The objective is to provide users with a clear and immediate indication of task that is scheduled at the present moment.

**Describe the solution you'd like**
Incorporate a visually distinctive element, such as color highlighting or a small icon, to clearly signify tasks scheduled for the current time.

**Comment by tijlleenders** [comment]
Yeah, this is useful I think. We could make the color bar introduced on the edge of the screen in #1766 stick out a little?

**Comment by tijlleenders** [comment]
Blocked by #1766
